plugins/configuration.rb

```diff
@@ -83,14 +85,37 @@ def render_config_vars(vars:, component:, platform:, converter:, classes: nil, p
             unless ['list', 'template'].include? parent_type
         end
 
+        defaultValue = ""
+        isDefault = false
+        if attr.key? 'default' and not attr['default'].to_s.empty? 
+          isDefault = true
+          defaultValue = converter.convert(attr['default'].to_s)
+        elsif attr['type'].to_s.include? 'boolean'
+          # Is the type is a boolean, a default key is required
```

Comment:
Not your fault, but easy fix:

```suggestion
          # If the type is a boolean, a default key is required
```